modules/apps/document-library/document-library-uad-test/src/testIntegration/java/com/liferay/document/library/uad/exporter/test/DLFileEntryUADExporterTest.java

```diff
@@ -77,6 +81,19 @@ public void testExportAll() throws Exception {
 		Assert.assertEquals(entries.toString(), 2, entries.size());
 	}
 
+	@ExpectedLogs(
```

Comment:
when I run the test the test fails and this is the log:

Can you take a look?

```
com.liferay.document.library.uad.exporter.test.DLFileEntryUADExporterTest > testExportAllWithMissingBinary FAILED
    java.lang.AssertionError: null
    	at org.junit.Assert.fail(Assert.java:89)
    	at com.liferay.portal.test.rule.LogAssertionTestRule.endAssert(LogAssertionTestRule.java:97)
    	at com.liferay.portal.test.rule.LogAssertionTestRule.afterClass(LogAssertionTestRule.java:186)
    	at com.liferay.portal.test.rule.LogAssertionTestRule.afterMethod(LogAssertionTestRule.java:193)
    	at com.liferay.portal.test.rule.LogAssertionTestRule.afterMethod(LogAssertionTestRule.java:50)
    	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:106)
    	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
    	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
    	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
    	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$1.evaluate(TestExecutorRunnable.java:237)
    	at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
    	at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
    	at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
    	at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
    	at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
    	at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
    	at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
    	at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
    	at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
    	at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
    	at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
    	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
    	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable._execute(TestExecutorRunnable.java:316)
    	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable.run(TestExecutorRunnable.java:95)
    	at java.lang.Thread.run(Thread.java:748)
```

Reply:
Fixed.